packages/terra-grid/src/Grid.scss

```diff
@@ -4,44 +4,47 @@
 
 // Grid Component
 // ========================================================
+/* stylelint-disable selector-class-pattern */
+:local {
+  .grid {
+    box-sizing: border-box;
+    display: flex;
+    flex-basis: auto;
+    flex-direction: row;
+    flex-grow: 0;
+    flex-shrink: 1;
+    flex-wrap: wrap;
+    margin-left: calc(-1*var(--terra-Grid-gutter-width));
```

Comment:
Using the https://github.com/postcss/postcss-custom-properties plugin, this compiles to `margin-left: calc(-1*1rem);`

This looks like its working in IE10. I'm not seeing a visual difference between what is in master and what is in this branch.
![ie-10-grid](https://user-images.githubusercontent.com/633148/28291738-56142d58-6b11-11e7-9e8f-1a77c416625c.gif)